aspnetcore/blazor/globalization-localization.md

```diff
@@ -0,0 +1,157 @@
+---
+title: ASP.NET Core Blazor globalization and localization
+author: guardrex
+description: Learn how to make Razor components accessible to users in multiple cultures and languages.
+monikerRange: '>= aspnetcore-3.1'
+ms.author: riande
+ms.custom: mvc
+ms.date: 02/12/2020
+no-loc: [Blazor, SignalR]
+uid: blazor/globalization-localization
+---
+# ASP.NET Core Blazor globalization and localization
```

Comment:
Consider adding an **Additional resources** section that links to https://docs.microsoft.com/en-us/aspnet/core/fundamentals/localization?view=aspnetcore-3.1